schemas/datasources/tempo/tempo.json

```
@@ -0,0 +1,7 @@
+{
+    "kind": "TempoDatasource",
+    "spec": {
+      "direct_url": "http://localhost:3200"
```

It should fix one error in the CI about `cue`

```suggestion
      "directUrl": "http://localhost:3200"
```

Then you should run `make fmt` and that should fix the rest of the errors

---

I ran make fmt, thanks for the tip!